templates/vue/src/components/node-modal/AppearanceForm.vue

```diff
@@ -102,8 +102,15 @@ export default {
       addLockedThumbnail: false,
     }
   },
+  watch: {
+    addThumbnail(addThumbnail) {
+      if (!addThumbnail) {
+        this.node.imageURL = ""
+      }
+    },
+  },
```

Comment:
Missing implementation for addLockedThumbnail

Reply:
Which may be the reason why I'm getting this issue:
1. External Link https://www.google.com/
2. Remove node appearance
3. Click add thumbnail
	- Link for "Show a different thumbnail when locked" auto-fills. When resaved, no UI changes.